qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/customers/CustomerDatabaseQueries.groovy

```diff
@@ -195,71 +197,99 @@ class CustomerDatabaseQueries {
         if (customerExists(customer)) {
             throw new DatabaseQueryException("Customer is already in the database.")
         }
-        int customerId = createNewCustomer(customer)
-        storeAffiliation(customerId, customer.affiliations)
+        Connection connection = databaseSession.getConnection()
```

Comment:
Same here, why is the connection created here and not provided as method parameter?

Reply:
see my previous answer please